pkg/filters/filters.go

```diff
@@ -465,6 +465,24 @@ func filterByPort(portStrs []string, getPort func(*v1.Event) (port uint16, ok bo
 	}, nil
 }
 
+func filterByReplyField(replyParams []bool) FilterFunc {
+	return func(ev *v1.Event) bool {
+		if len(replyParams) == 0 {
+			return true
+		}
+		if ev.GetFlow() == nil {
```

Comment:
Unfortunately, since #30 `GetFlow` returns an interface type and this check will never be true. We should probably add a `HasFlow()` to `v1.Event` for this purpose.

Reply:
rebased with #30 merged. added a type switch for `ev.Event.(type)` instead of checking if `GetFlow` is nil.